dbt/include/athena/macros/materializations/models/table/create_table_as.sql

```diff
@@ -90,41 +90,70 @@
 
 {% macro create_table_as_with_partitions(temporary, relation, sql) -%}
 
-    {% set partitions_batches = get_partition_batches(sql) %}
+    {%- set stg_relation = api.Relation.create(
+            identifier=relation.identifier ~ '__stg',
```

Comment:
we generally call those relation `tmp_relation` see https://github.com/dbt-athena/dbt-athena/blob/main/dbt/include/athena/macros/materializations/models/table/table.sql#L21-L25 just for consistency of naming I will suggest to pick `tmp_relation` and eventually consider to use `__tmp` as prefix, or something like 
`__tmp_no_partitions` to make even more explicitly the context of that tmp table name

Reply:
set to `__tmp_not_partitioned`